tests/VersionParserTest.php

```diff
@@ -66,11 +66,13 @@ public function successfulNormalizedVersions()
             'parses long/2' => array('10.4.13beta2', '10.4.13.0-beta2'),
             'parses long/semver' => array('10.4.13beta.2', '10.4.13.0-beta2'),
             'expand shorthand' => array('10.4.13-b', '10.4.13.0-beta'),
-            'expand shorthand2' => array('10.4.13-b5', '10.4.13.0-beta5'),
+            'expand shorthand/2' => array('10.4.13-b5', '10.4.13.0-beta5'),
             'strips leading v' => array('v1.0.0', '1.0.0.0'),
+            'parses dates y-m as classical' => array('2010.01', '2010.01.0.0'),
+            'parses dates w/ . as classical' => array('2010.01.02', '2010.01.02.0'),
```

Comment:
Is this really parsed as classical version if the leading zeroes remain in .01 and .02?